### PR TITLE
Make Sequelize work with TypeScript@next

### PR DESCRIPTION
**Pull request by kant2002**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

Without this change I could not make work that code when install it using @types/sequelize

### CODE REVIEW

**Comment by mhegazy** [comment]
Thank you for your contribution. 

Can you please fix the failing Travis CI tests and update the PR!

File: sequelize/index.d.ts [hunk 1]
```diff
@@ -6160,4 +6160,6 @@ declare namespace sequelize {
 
 declare var sequelize: sequelize.SequelizeStatic;
 
-export = sequelize;
+declare module "sequelzie" {
```
Comment (mhheise) [inline sequelize/index.d.ts]: Should this be `declare module "sequelize" {` instead? This might have been a simple typo.

Comment (kant2002) [inline sequelize/index.d.ts]: Thanks

**Comment by kant2002** [comment]
@mhegazy I file issue agains https://github.com/DefinitelyTyped/definition-tester/issues/19 where I take a first stab on making dt tool work for types-2.0 branch. @RyanCavanaugh, if this is correct difrection to take in making dt tool works, I will continue forward and made full PR with implementing changes.

**Comment by mhegazy** [comment]
> Without this change I could not make work that code when install it using @types/sequelize

what version of the TS compiler were you using? and what are the errors you get?

I do not think this is the correct change, as it adds a named module in the global namespace, which will result into conflicts in the future if two different versions of the module were loaded.

**Comment by kant2002** [comment]
@mhegazy it was latest nightly version of TypeScript available as of day I post issue. Let me check with TypeScript@beta and if something still produce error, I will create small repro.

**Comment by mhegazy** [comment]
The CI server should be working now, can you refresh the PR?

**Comment by paulvanbrenk** [comment]
@kant2002 what are you specifically trying to fix. the version that's currently in the 2.0 branch compiles and seems to work as expected?

**Comment by kant2002** [comment]
@paulvanbrenk I was trying to fix instalation via NPM module `@types/sequelize`. Older version was not working. I did not check for a while what is the current state.

**Comment by paulvanbrenk** [comment]
@kant2002 If you have time can you check if the current checked in version works as expected for you and close this PR if it does?

Thanks for helping out!

**Comment by kant2002** [comment]
It is working @paulvanbrenk in current version from NPM, Thanks!

**Pull request closed without merging**